spec/forms/hyrax/forms/resource_form_spec.rb

```diff
@@ -438,7 +438,7 @@
 
     context 'when using a generic valkyrie adapter', valkyrie_adapter: :test_adapter do
       before do
-        allow(Hyrax).to receive_message_chain(:config, :disable_wings).and_return(true) # rubocop:disable RSpec/MessageChain
+        allow(Hyrax.config).to receive(:disable_wings).and_return(true)
```

Comment:
This was needed because the indexing done in the factory uses Hyrax.config which was not allowed in this stubbing.

Reply:
This is a cleaner stub.